### PR TITLE
Update interpreter.h to avoid error on windows

### PR DESCRIPTION
**Pull request by SagaraBattousai**
Previous code caused error C2665 on windows: 'std::atomic_flag::atomic_flag': no overloaded function could convert all the argument types

Instead of assigning with a bool use ATOMIC_FLAG_INIT for the same effect

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @zichuan-wei, Can you please review this PR ? Thank you!

File: tensorflow/lite/CMakeLists.txt [hunk 1]
```diff
@@ -98,6 +98,8 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
   # 2b96f3662bd776e277f86997659e61046b56c315/tensorflow/lite/tools/make/\
   # Makefile#L157
   set(_TFLITE_ENABLE_MMAP OFF)
+  #Should this also be true if TFLITE_ENABLE_MMAP=OFF even if windows is not the system name?
```
Comment (zichuan-wei) [inline tensorflow/lite/CMakeLists.txt:101]: Thanks for submitting the fix

Comment (zichuan-wei) [inline tensorflow/lite/CMakeLists.txt:101]: Given that when the system is windows, MMAP already set to off, is this compile option necessary?

**Comment by gbaned** [comment]
Hi @SagaraBattousai Can you please check @zichuan-wei's comments and keep us posted? Thank you!

**Comment by gbaned** [comment]
Hi @SagaraBattousai Any update on this PR? Please. Thank you!

**Pull request closed without merging**